.github/workflows/ci.yml

```diff
@@ -27,28 +28,28 @@ jobs:
       shell: bash
 
     - name: Run integrity checks
-      env:
-        DATATRAILS_URL: ${{ vars.DATATRAILS_URL }}
-        DATATRAILS_CLIENT_ID: ${{ vars.DATATRAILS_CLIENT_ID }}
-        DATATRAILS_CLIENT_SECRET: ${{ secrets.DATATRAILS_CLIENT_SECRET }}
-
       run: |
         ruff check datatrails_scitt_samples unittests
         python3 -m black datatrails_scitt_samples unittests
+        python3 -m pyright --stats datatrails_scitt_samples
         modified=$(git status -s | wc -l)
         if [ $modified -gt 0 ]
         then
             echo "there are $modified files that must be reformatted"
             echo "DISABLED guard due to mismatch with local environment"
             # exit 1
         fi
-        python3 -m unittest
       shell: bash
-    - name: Run type-hint checks
-      if: ${{ matrix.python-version != '3.12' }}
+
+    - name: Unit tests
```

Comment:
very minor: Do we really want to call these unit tests ? It is true they are being run using the unittest framework, but they are actually *end to end* tests for the whole scitt solution on our platform. If there is a consistent thing to do here lets do that. but its small beer

Reply:
yes - I think we should rename them

Reply:
renamed to 'tests'